docs/api/composables.md

```diff
@@ -209,3 +207,19 @@ import { useTresContext } from '@tresjs/core'
 const context = useTresContext()
 </script>
 ```
+
+### Properties of context
+| Property | Description |
+| --- | --- |
+| **camera** | the currently active camera |
+| **cameras** | the cameras that exist in the scene |
+| **controls** | the controls of your scene |
+| **deregisterCamera** | a method to deregister a camera. This is only required if you manually create a camera. Cameras in the template are deregistered automatically. |
+| **extend** | extends the component catalogue|
```

Comment:
Here we can reference the extend docs https://tresjs.org/advanced/extending.html

Reply:
I added that.